docs/content/_index.md

```diff
@@ -9,7 +9,8 @@ Everything you need to know about Brigade.
 
 If you haven't used Brigade before, welcome! 🎉
 Brigade 2.0 has a new design that makes learning and interacting with Brigade easier than ever before.
-It is now a first-class experience that doesn't require familiarity with Kubernetes, so you can quickly be productive without a steep learning curve.
+Brigade now has a first-class end-user experience that doesn't require familiarity with Kubernetes, so you can quickly be productive without a steep learning curve.
+Since it is still an application installed on Kubernetes, administrators, and those installing Brigade, will still need to know how to install and configure applications on Kubernetes.
```

Comment:
Besides Brigade's control plane being installed on k8s, Brigade _uses_ k8s as a "workload execution substrate" as well. Is that worth saying? I honestly don't know. It's just a matter of how much do we want to drive the point home that k8s is a _dependency_, but once that's satisfied, you can forget about it. 🤷

Reply:
This is just the welcome page for the docs, and I think that's a bit too much info too soon. How about I undo my changes here and save any comparison of v1 and v2 for a blog post and the migration page instead?

Reply:
Maybe it is best not to promise "less k8s" and then dive right away into installing... on k8s.

But really, I'm fine with a wide range of possibilities here. Whatever you think is best.

I'm just thinking out loud. Sorry!

Reply:
I've reverted changes to this page since I don't want the quickstart to get sidetracked by this change.